python/py-owslib/Portfile

```diff
@@ -25,24 +25,26 @@ homepage            https://geopython.github.io/OWSLib/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
```

Comment:
these are the defaults for "master_sites" and "distname" as set by the "python" PG so they can be removed

Reply:
Fixed, thanks!